docs/backend-requests/making/jwt-templates.mdx

```diff
@@ -160,7 +160,74 @@ For example, you could use interpolation to build the user's full name:
 }
 ```
 
-Interpolated shortcode substituted values will always be casted to strings. For example, if the user does not have a last name associated, the above full name value would be "null John".
+Interpolated shortcodes will always result to string values. For example, if the user does not have a last name associated, the above full name value would be "null John".
```

Comment:
@agis Can we make it so that the strings are always empty instead of `null`? It's better UX for our customers to get "John" than "null John". Also notice that the value is trimmed for whitespaces.

Reply:
Note that this paragraph is pre-existing and refers to interpolation, not conditional expressions. It just appeared changed here because I re-formatted it due to max column limits.

That said, we can certainly revisit interpolation and change the way it works. However, do note that with conditional expressions now one can do:

```json
{{ user.last_name || '' }} {{ user.first_name || ''}}
```